client/packages/common/src/authentication/api/hooks/useLogin.ts

```diff
@@ -58,6 +58,27 @@ const getMostRecentCredentials = (
   return [];
 };
 
+// returns MRU store, if set
+// or the first store in the list
+export const getStore = async (
```

Comment:
Moved out of login function, so that store preferences could be updated when a user is updated